src/themes/five/src/components/SingleProductCard/index.js

```diff
@@ -129,6 +129,7 @@ const SingleProductCardUI = (props) => {
         isCartOnProductsList={isCartOnProductsList}
         style={useCustomFunctionality && customStyle}
         className='product-card'
+        hasAddButt={!useCustomFunctionality && typeof hideAddButton !== 'undefined' && !hideAddButton && !isSkeleton}
```

Comment:
Please rename this prop `hasAddButt` appropriately (`isHideAddButton` for example), and fix the condition, you're doing unnecesaries validations for example `typeof hideAddButton !== 'undefined'`